lesson 4/Tensor/Tensor/Exception.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+
+class Exception {
```

Comment:
Your exception must derive from std::exception and override  virtual const char* std::exception::what()

Reply:
And it should have more explicit name like WrongShapeTensorException, not just exception